File: android/src/main/kotlin/com/webengage/we_personalization_flutter/model/WEProperty.kt
package com.webengage.we_personalization_flutter.model

import com.webengage.personalization.data.WECampaignData

data class WEProperty(
    val id: Int,
    val screenName: String,
    val propertyID: String
) {
    var weCampaignData: WECampaignData? = null
}

File: ios/Classes/model/WEProperty.swift
import WEPersonalization
class WEProperty{
    var id:Int
    var screenName:String
    var propertyID:Int
    var campaignData : WECampaignData? = nil
    
    public init(id: Int, screenName: String, propertyID: Int) {
        self.id = id
        self.screenName = screenName
        self.propertyID = propertyID
    }
    
    
    
    
}
